code-challenges.js

```diff
@@ -13,23 +14,54 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+const oddNums = (array) => {
+    let newArr = array.filter(value => {
+      if(typeof value === "number"){
+        return value % 2 !== 0
+      }
+    })
+    return newArr.sort((a, b) => a - b)
+}
 
+console.log(oddNums(fullArr1))
+console.log(oddNums(fullArr2))
 
 
 // --------------------3) Create a function that takes in a string of a single word and returns the middle letter of the word. If the word is an even number of letters, return the two middle letters.
 
 var middleLetters1 = "hello"
-// Expected output: “l”
+//Expected output: “l”
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
-
-
-
+var position = ""
+var length = 0
+
+const midLetter = (str) => {
+  if(str.length() % 2 == 1){
+  let position = str.length() / 2
+  let  length = 1
+  }
+  return string.substring(position, postion + length)
+}
```

Comment:
this code threw a lot of errors just to get it running. did you run this in your terminal?